### ISSUE TITLE
Always return Stripe\Card when calling Stripe\Token->card

### ISSUE THREAD

**Issue by ruudk** [stripe/stripe-php#402]
Currently, it returns an `AttachedObject`, while I think it should return a `Card`

**Comment by ruudk** [comment]
I now have to do this:
```php
public function getCard(string $token) : Card {
    $token = Token::retrieve($token);
    return Util::convertToStripeObject($token->card->__toArray(), null);
}
```

and that looks very ugly :D

**Comment by ob-stripe** [comment]
Hi @ruudk. This is a known issue described [here](https://github.com/stripe/stripe-php/pull/393#discussion_r153878518). We have a fix pending in #396 but it's turning out to be slightly more complex than anticipated.

We're still going to fix this, but OOC, why do you need the `card` property to be a card object?
